### ISSUE TITLE
Develop a tuning toolkit

### ISSUE THREAD

**Issue by PonteIneptique** [hipster-philology/pandora#81]
General idea : 

- [ ] tuning.py based on pandora.cli.tuning_cmd
- [ ] Runs a series of configs directly from the command line
- [ ] Loads a list of configs from a folder through --configs_dir configs/*.txt
- [ ] Keep the last evaluation score in a csv using the name of the config as a key
- [ ] Produce a CSV with learning curve to adapt where necessary the learning rate and other params

**Comment by PonteIneptique** [comment]
Other idea : 

The actual tuning.py could have a second option to generate such configs using a csv file : 

```csv
name,include_lemma,include_pos,nb_left_tokens...
config1,True,True,2
config2,True,True,1
```

**Comment by emanjavacas** [comment]
Given a dev set there is nothing stopping us from doing hyperparameter
optimization. There are a couple of non-bayesian methods worth trying and
offering them for people with GPUs and time to invest in squeezing the most
out of their data. Things like hyperband or successive halving.

**Comment by PonteIneptique** [comment]
I am all for method-based hyperparameter optimization. The only thing is that it might take more time that just what I described. But please, if you find the time to do this one, I'd be glad to test it :)

**Comment by PonteIneptique** [comment]
Note : we could also simply use existing libraries such as https://github.com/hyperopt/hyperopt
